bin/templates/templates.xml

```diff
@@ -251,4 +251,26 @@
             </ul>
         ]]></description>
     </template>
+    <template isTestPlan="true">
+        <name>Simple HTTP request</name>
+        <fileName>/bin/templates/simple-http-request-test-plan.jmx</fileName>
+        <description><![CDATA[
+        <h1>Template for simple HTTP request</h1>
+        <h2>JMeter Configuration</h2>
+        <p>Configure your HTTP request, set:
+            <ul>
+                <li>Full URL</li>
+                <li>Content Type</li>
+				<li>HTTP method</li>
+				<li>Body data</li>
+            </ul>
+        </p>
+        ]]></description>
+        <parameters>
+            <parameter defaultValue="https://jmeter.apache.org/" key="url"/>
```

Comment:
Maybe we should take another domain than our own, so no DOS will result ;) In the documentation we use something ending on `example.invalid`. That way no one innocent will be harmed by the default settings.